src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -271,6 +271,17 @@ internal void Validate()
                     "See documentation on Task Hubs for information on how to set this: https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-task-hubs");
             }
 
+            string runtimeLanguage = environmentVariableResolver.Resolve("FUNCTIONS_WORKER_RUNTIME");
+            if (!string.Equals(runtimeLanguage, "dotnet", StringComparison.OrdinalIgnoreCase))
```

Comment:
Shouldn't we also be checking if `this.ExtendedSessionsEnabled == true`? Otherwise, won't we always log this for every non-.NET function app even if they set it to false?

Reply:
Ah, good catch!

Reply:
Also, should we consider the case where this environment variable is not set at all? There are many .NET apps which don't set this environment variable, and I'm worried that we'd be inadvertently disabling extended sessions for those apps that upgrade to this extension version.